pages/book-table.html

```diff
@@ -3,7 +3,108 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Document</title>
+    <meta
+      name="author"
+      content="Greta Masliukova, Ricardo Vicente, Ilona Liashenko, Ielyzaveta Shabetnyk, Group2"
+    />
+    <meta name="description" content="The Pink Bear restaurant" />
+    <meta
+      name="keywords"
+      content="Restaurant, Pink Bear, food, food menu, services, website"
+    />
+    <link rel="stylesheet" href="../css/normalize.css" />
+    <link rel="stylesheet" href="../css/styles.css" />
+    <link rel="stylesheet" href="../css/book-table.css" />
+    <title>Book Table</title>
   </head>
-  <body></body>
+  <body>
+    <nav id="page-navigation" class="main-nav">
+      <ul>
+        <li>
+          <a href="../index.html"
+            ><img
+              src="../assets/svg/paw-solid.svg"
+              alt="paw-logo"
+              class="paw-logo"
+          /></a>
+        </li>
+        <div class="main-nav__flex">
+          <li><a href="../pages/about.html" class="f-s36">About us</a></li>
+          <li>
+            <a href="../pages/menu.html" class="f-s36">Menu</a>
+          </li>
+          <li>
+            <a href="../pages/application.html" class="f-s36">Careers</a>
+          </li>
+          <li>
+            <button class="primary-btn mr-60">
+              <a href="../pages/book-table.html" class="caps-text f-s32"
+                >Order now</a
+              >
+            </button>
+          </li>
+        </div>
+      </ul>
+    </nav>
+    <header></header>
+    <main>
+      <section class="book-table">
+        <h1 class="book-table__title ff-arial f-s40 fw700">Book Table</h1>
+        <form class="book-table__form">
+          <label class="ff-arial f-s36 fw400" for="first name"
+            >First Name</label
+          >
+          <input
+            class="book-table__input-firstname"
+            type="text"
+            name="first name"
```

Comment:
pls check the name attribute. it should not be separated with space if there are 2 words rather a hyphen. 
    "Use lowercase Latin letters.
    Separate different words by a hyphen.
    Do not use periods as well as numbers at the beginning of the name."